### ISSUE TITLE
Allow the user to change the background colour

### ISSUE THREAD

**Issue by martinburchell** [ucam-department-of-psychiatry/camcops#215]
A CPFT patient with dyslexia needs a blue background in order to read. We could make the colours configurable but would need to do this in a way that kept the meaning of certain colours, such as "this bit needs to be filled in by a clinician".

**Comment by RudolfCardinal** [comment]
Yes. Good idea. Probably need to be able to set all colours from a menu, then (and maybe have preset combinations). Will be things like "menu background", "questionnaire background (for completion by subject)", "questionnaire background (for completion by clinician/researcher)"...
